pkg/cmd/dev/test.go

```diff
@@ -105,12 +110,20 @@ pkg/kv/kvserver:kvserver_test) instead.`,
 	testCmd.Flags().String(testArgsFlag, "", "additional arguments to pass to the go test binary")
 	testCmd.Flags().String(vModuleFlag, "", "comma-separated list of pattern=N settings for file-filtered logging")
 	testCmd.Flags().Bool(showDiffFlag, false, "generate a diff for expectation mismatches when possible")
+	testCmd.Flags().String(volumeFlag, "bzlhome", "the Docker volume to use as the container home directory (only used for roachprod-stress cross builds)")
```

Comment:
Here and below, refers to roachprod-stress. How about the following instead?

- the docker volume to use for cross builds (applies to --stress --cluster)
- roachprod cluster to use when stressing remotely